pepper-apis/src/main/java/org/broadinstitute/ddp/DataDonationPlatform.java

```diff
@@ -240,51 +242,27 @@ private static void start() {
         if (cfg.hasPath(ConfigFile.PREFERRED_SOURCE_IP_HEADER)) {
             preferredSourceIPHeader = cfg.getString(ConfigFile.PREFERRED_SOURCE_IP_HEADER);
         }
-        JettyConfig.setupJetty(preferredSourceIPHeader);
-
-        if (appEnginePort != null) {
-            port(Integer.parseInt(appEnginePort));
-        } else {
-            port(configFilePort);
-        }
 
         String dbUrl = cfg.getString(ConfigFile.DB_URL);
         LOG.info("Using db {}", dbUrl);
 
         TransactionWrapper.init(
                 new TransactionWrapper.DbConfiguration(TransactionWrapper.DB.APIS, maxConnections, dbUrl));
-
-        threadPool(-1, -1, requestThreadTimeout);
         Config sqlConfig = ConfigFactory.load(ConfigFile.SQL_CONFIG_FILE);
         initSqlCommands(sqlConfig);
 
-        if (doLiquibase) {
-            LOG.info("Running liquibase migrations against " + dbUrl);
-            LiquibaseUtil.runLiquibase(dbUrl, TransactionWrapper.DB.APIS);
+        JettyConfig.setupJetty(preferredSourceIPHeader);
+        if (appEnginePort != null) {
+            port(Integer.parseInt(appEnginePort));
```

Comment:
It might be worth moving the `port()` call up a bit--the faster we hit that call, the sooner we can block on `ah/start` and avoid premature GAE traffic.